travel-agency-web/src/main/java/cz/muni/fi/pa165/bluebat/controllers/ReservationController.java

```diff
@@ -0,0 +1,48 @@
+package cz.muni.fi.pa165.bluebat.controllers;
+
+import cz.muni.fi.pa165.bluebat.ApiUris;
+import cz.muni.fi.pa165.bluebat.dto.ReservationCreateDTO;
+import cz.muni.fi.pa165.bluebat.dto.ReservationDTO;
+import cz.muni.fi.pa165.bluebat.dto.TripCreateDTO;
+import cz.muni.fi.pa165.bluebat.dto.TripDTO;
+import cz.muni.fi.pa165.bluebat.facade.ReservationFacade;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * @author : Rudolf Madoran
+ * @since : 22. 5. 2021, Sat
+ **/
+
+@RestController
+@RequestMapping("/main")
+public class ReservationController {
+
+    @Autowired
+    private ReservationFacade reservationFacade;
+
+    @RequestMapping(value = ApiUris.ROOT_URI_RESERVATIONS+"/create", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE,
```

Comment:
ještě nemusíš to dělat takto složitě, stačí jen na 19. řádku nastavit "/reservation" a tady pak dáš už jen "/create" -> pak ti to vytvoří API url "pa165/rest/reservation/create". Kdežto teď to máš "pa165/rest/main/reservation/create"